doc/Getting-started/index.md

```diff
@@ -54,40 +39,66 @@ You can either install Kaa Sandbox or run it on Amazon EC2.
 
 <br>
 
+To run the Kaa Sandbox, your system must meet the following minimum requirements:
+
+- 64-bit OS
+- 4 GB RAM
+- Virtualization enabled in BIOS
+
 To install Kaa Sandbox, perform the following steps:
 
-1. Install the virtualization environment. 
-The current version of Kaa Sandbox supports [Oracle VirtualBox 4.2+](https://www.virtualbox.org/wiki/Downloads) which is available as a free download.
+1. Install the virtualization environment.
+The current version of Kaa Sandbox supports Oracle VirtualBox 4.2+, which is available as a [free download](https://www.virtualbox.org/wiki/Downloads).
 
-2. Download the Sandbox image from [Kaa download page](http://www.kaaproject.org/download-kaa/).
+2. Download the Sandbox .ova image from the [Kaa download page](http://www.kaaproject.org/download-kaa/).
 
-3. Import the Sandbox image using this [guide](https://www.virtualbox.org/manual/ch01.html#ovf).
+3. Import the Sandbox image using [this guide](https://www.virtualbox.org/manual/ch01.html#ovf).
 
-<br>
+4. Once the image has loaded, open the machine settings, select the `System` tab, and adjust the amount of RAM and processors available to the VM.
+For optimal performance we recommend at least 4 GB of RAM, and at least 2 CPUs.
 
-</div><div id="Amazon" class="tab-pane fade" markdown="1">
+5. *\[Optional\]* The default VM networking configuration is NAT.
+In this configuration you will only be able to connect to the Kaa server from your host machine.
+That includes the Kaa client applications, which will only be able to reach your Sandbox server from the host machine.
+
+   Should you wish to deploy client applications to any hardware other than your host machine, or access Kaa web interfaces from your networked devices, the networking mode should be changed to `Bridged Adapter`.
+To configure the networking mode, switch to the `Network` tab in the VM settings.
+Make sure you change the `Promiscuous Mode` to `Allow All` under the `Advanced` spoiler.
+
+   ![VM Networking settings](attach/vm_networking_settings.png)
+
+6. Apply the VM settings and start the virtual machine.
+
+7. Wait until the machine boots up completely and open **127.0.0.1:9080/sandbox** URL in your browser.
+(If you have changed the networking mode to Bridged Adapter, you should rather use **<host\_machine\_ip>:9080/sandbox**.)
 
 <br>
 
-To launch the Kaa sanbox on Amazon Elastic Compute Cloud (Amazon EC2), go through the following steps.
+</div><div id="EC2" class="tab-pane fade" markdown="1">
 
-1. Launch the AMI using the links in the following table:
+<br>
 
-   Amazon EC2 offers a number of [geographic regions](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) for launching the AMI. Factors for choosing a region include: reduce latency, cost, or regulatory requirements.
+> **NOTE:** By running your Kaa Sandbox instance on AWS EC2 you may incur the associated charges.
```

Comment:
I think it's a good idea to use quotation for notes and warning - markdown doesn't provide formatting elements for this.